source/js/buyers-guide/related-articles.js

```diff
@@ -0,0 +1,21 @@
+const RelatedArticles = {
+  init: () => {
+    const firstParagraph = document.querySelector(
+      ".paragraph-block .rich-text"
+    );
+    const relatedContainer = document.querySelector("#pni-related-content");
+    if (!firstParagraph || !relatedContainer) return;
+    const relatedContent = relatedContainer.querySelector("div");
+    const relatedPlace = firstParagraph.querySelector("*:nth-child(3)");
+    if (!relatedPlace) return;
```

Comment:
Can you add some context on what is happening here?

Reply:
this basically checks if there is at least 3 children inside the rich body container. Usually there is a header(h1-6) and a few paragraphs blocks to start most articles. If we do not have at least that many elements we leave the related articles at the bottom of the article page.

Reply:
What do you think about adding this as a comment or docstring to the method?